<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interval_picker</class>
 <widget class="QDialog" name="interval_picker">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>471</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Week:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="week_label">
        <property name="maximumSize">
         <size>
          <width>228</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="prev_week">
        <property name="maximumSize">
         <size>
          <width>37</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="next_week">
        <property name="maximumSize">
         <size>
          <width>37</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="QPushButton" name="actual_week">
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Current Week</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Month:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="month_label">
        <property name="maximumSize">
         <size>
          <width>228</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="prew_month">
        <property name="maximumSize">
         <size>
          <width>37</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="next_month">
        <property name="maximumSize">
         <size>
          <width>37</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="firstDayOfWeek">
       <enum>Qt::Monday</enum>
      </property>
      <property name="gridVisible">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QCalendarWidget::SingleSelection</enum>
      </property>
      <property name="horizontalHeaderFormat">
       <enum>QCalendarWidget::ShortDayNames</enum>
      </property>
      <property name="navigationBarVisible">
       <bool>true</bool>
      </property>
      <property name="dateEditEnabled">
       <bool>true</bool>
      </property>
      <property name="dateEditAcceptDelay">
       <number>1000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>buttonBox</zorder>
  <zorder>label</zorder>
  <zorder>calendarWidget</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>prev_week</zorder>
  <zorder>next_week</zorder>
  <zorder>actual_week</zorder>
  <zorder>prew_month</zorder>
  <zorder>next_month</zorder>
  <zorder>actual_week</zorder>
  <zorder>actual_week</zorder>
  <zorder>next_week</zorder>
  <zorder>prev_week</zorder>
  <zorder>next_month</zorder>
  <zorder>prew_month</zorder>
  <zorder>actual_week</zorder>
  <zorder>week_label</zorder>
  <zorder>month_label</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>interval_picker</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>interval_picker</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
